app/views/maths_and_physics/claims/eligibility_confirmed.html.erb

```diff
@@ -4,5 +4,7 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">You are eligible to claim a payment for teaching maths or physics</h1>
     <p class="govuk-body">Based on your answers you can claim £2,000 for teaching maths or physics.</p>
+
+    <%= link_to "Continue", claim_path(current_policy_routing_name, next_slug), class: "govuk-button", role: "button" %>
```

Comment:
Does this need `data: { module: "govuk-button" }` as it is a link that has button behaviour?

Reply:
👍🏽